aiogram/bot/base.py

```diff
@@ -173,6 +174,8 @@ def with_token(self, bot_token: base.String, validate_token: Optional[base.Boole
         finally:
             self._ctx_token.reset(token)
 
+    @deprecated("This method behaviour will be changed in aiogram v3.0. "
```

Comment:
```suggestion
    @deprecated("This method's behavior will be changed in aiogram v3.0. "
```

biased "behavior" word change, anyways can keep it unchanged